Neolution.Extensions.Identity/JwtSignInManager.cs

```diff
@@ -0,0 +1,116 @@
+﻿namespace Neolution.Extensions.Identity
+{
+    using System.Security.Claims;
+    using Google.Apis.Auth;
+    using Microsoft.AspNetCore.Identity;
+    using Neolution.Extensions.Identity.Abstractions;
+    using JsonWebToken = Neolution.Extensions.Identity.Abstractions.JsonWebToken;
+
+    /// <inheritdoc/>
+    public sealed class JwtSignInManager<TUser> : IJwtSignInManager<TUser>
+        where TUser : IdentityUser<Guid>
+    {
+        /// <summary>
+        /// The user manager
+        /// </summary>
+        private readonly IUserManager<TUser> userManager;
+
+        /// <summary>
+        /// The sign in manager
+        /// </summary>
+        private readonly ISignInManager<TUser> signInManager;
+
+        /// <summary>
+        /// The JWT generator
+        /// </summary>
+        private readonly IJwtGenerator<TUser> jwtGenerator;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="JwtSignInManager{TUser}"/> class.
+        /// </summary>
+        /// <param name="userManager">The user manager.</param>
+        /// <param name="signInManager">The sign in manager.</param>
+        /// <param name="jwtGenerator">The JWT generator.</param>
+        public JwtSignInManager(IUserManager<TUser> userManager, ISignInManager<TUser> signInManager, IJwtGenerator<TUser> jwtGenerator)
```

Comment:
Should we call `AccessFailedAsync` somewhere in this method in case the login fails. Or does it not make any sense to support lockout for external providers?

Reply:
As discussed, we leave it out for the moment